src/seml/config.py

```diff
@@ -343,10 +341,13 @@ def _sacred_create_configs(exp: 'sacred.Experiment', configs: List[Dict], named_
     from sacred.utils import convert_to_nested_dict, recursive_update, iterate_flattened, join_paths, set_by_dotted_path
     from sacred.initialize import (create_scaffolding, gather_ingredients_topological, distribute_config_updates, 
                                    get_configuration, get_scaffolding_and_config_name, distribute_presets)
+    from tqdm import tqdm
+    
     configs_resolved = []
     if named_configs is None:
         named_configs = [()] * len(configs)
-    for config, named_config in zip(configs, named_configs):
+    for config, named_config in tqdm(list(zip(configs, named_configs)), desc='Resolving configurations', 
```

Comment:
One could also use [rich's progress bar](https://rich.readthedocs.io/en/stable/progress.html), which is visually more coherent. no big feelings here ^^

Reply:
Done.